keps/sig-network/3453-minimize-iptables-restore/README.md

```diff
@@ -280,7 +261,7 @@ ways.
 
 ###### How can this feature be enabled / disabled in a live cluster?
 
-- [ ] Feature gate (also fill in values in `kep.yaml`)
+- [X] Feature gate (also fill in values in `kep.yaml`)
```

Comment:
Can't comment on unchanged lines, so commenting here:

L290: I would additionally add that this feature doesn't result in any API writes, so we don't need to validate this.

Reply:
L328: Have those metrics been added? If so, please link them. If not, we should add them.